mandatory/2-function-creation.js

```diff
@@ -25,7 +35,19 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
```

Comment:
This works a treat for the test case, well done. 

It's worth noting that `replace("/", "")` will only replace the first forward slash, if you have multiple that could be a problem. This still passes the test and I think future proofing this is a little more advanced and hasn't been covered in class yet so we can skip it.

I would consider your variable names, they don't make much sense and whilst this function is only 9 lines, imagine if it was 100 lines - it could become difficult to understand what each variable is meant to hold.

I would suggest more descriptive names like such:
```javascript
function tidyUpStrings(arrayOfStrings) {
  let formattedStrings = [];
  let newString = "";
  
  arrayOfStrings.forEach(function (string) {
      newString = string.trim().replace("/", "").toLowerCase();
      formattedStrings.push(newString);
  });
  
  return formattedStrings;
}